src/components/settings/SettingsLMSTab/LMSConfigs/CanvasConfig.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Form, useToggle } from '@edx/paragon';
+import { buttonText, handleErrors } from '../LMSConfigPage';
+import LmsApiService from '../../../../data/services/LmsApiService';
+import { snakeCaseDict } from '../../../../utils';
+import ConfigError from '../ConfigError';
+
+const CanvasConfig = ({ id, onClick }) => {
```

Comment:
I think we should rename `onClick` prop to something else more apt? the onClick-ness makes sense in the calling component.code, but here it looks like it's more like a `onSuccess` or sth like that?

Reply:
or id, sth like onCancel? not sure what the best name will be

Reply:
The only thing is it actually functions in multiple ways (as an onSubmit or an onCancel) so that's why its encompassed in click